react/Breadcrumb.tsx

```diff
@@ -70,7 +70,7 @@ class Breadcrumb extends Component<Props> {
             <span className="ph2">
               <ArrowIcon />
             </span>
-            <Link className={LINK_CLASS_NAME} to={`/${value}`}>
+            <Link className={`${LINK_CLASS_NAME} link link--visited`} to={`/${value}`}>
```

Comment:
Is link--visited really necessary? Since CSS has a selector to detect this?

Reply:
There's a discussion in the document about it, but since it's not on tachyons I think it must be removed, the client could use the selector you mentioned.